Beam/resources/assets/js/remplib.js

```diff
@@ -119,6 +132,19 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
             window.addEventListener("campaign_showtime", this.syncOverridableFields);
             window.addEventListener("campaign_showtime", this.syncFlags);
             window.addEventListener("beam_event", this.incrementSegmentRulesCache);
+
+            if (this.config.tracker.readingProgress.enabled) {
+
+                setInterval(function () {
+                    this.sendTrackedProgress()
```

Comment:
This causes `Uncaught TypeError: this.sendTrackedProgress is not a function` (in the context of callback, `this` doesn't point where you might think it points to)